sammo.py

```diff
@@ -110,6 +114,27 @@ def createSessionAction(self) -> SammoSessionAction:
     def initGui(self):
         pass
 
+    def initShortcuts(self) -> None:
+        self.environmentShortcut = QShortcut(
+            QKeySequence("Shift+E"), self.mainWindow
+        )
+        self.environmentShortcut.activated.connect(self.onEnvironmentAction)
+
+        self.followersShortcut = QShortcut(
+            QKeySequence("Shift+F"), self.mainWindow
+        )
+        self.followersShortcut.activated.connect(self.onFollowersAction)
+
+        self.sightingsShortcut = QShortcut(
+            QKeySequence("Shift+O"), self.mainWindow
+        )
+        self.sightingsShortcut.activated.connect(self.onSightingsAction)
+
+        self.sightingsShortcut = QShortcut(
```

Comment:
self.sightingsShortcut is already defined at line 128, shouldn't it be another attribute ?

Reply:
Yes indeed :)